http/RestCatalogLocalTest.http

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+### get catalog config
+GET http://localhost:1630/api/iceberg/rest/v1/config?warehouse=iceberg
```

Comment:
I am not sure what is the usage of this new file?

Reply:
This is for easy testing of HTTP interfaces in a local environment, and the 'HTTP Client' plugin can be installed in IDEA.

Reply:
So is this used for local testing? I'm not sure if this file is necessary for other developers. If it is, we might need to add a README to explain how to use it.

Additionally, I'm not sure if using the IDEA plugin is a reasonable way to test the REST API. Perhaps we should integrate them into unit tests or integration tests.

Reply:
This may not be suitable for unit tests or integration tests, as it relies on the local environment. I added a readme file to explain it.